src/prefect/client/schemas/objects.py

```diff
@@ -854,6 +855,28 @@ def __hash__(self):
         return hash(self.handle)
 
 
+class IPAllowlistEntry(PrefectBaseModel):
+    ip_network: IPvAnyNetwork
+    description: Optional[str] = Field(
+        default=None, description="A description of the IP entry."
+    )
+    enabled: bool
```

Comment:
nit: can we put `enabled` on top here? ie required and then optional fields

Reply:
u got it dude :thumbs-up: